frontend/src/utils/formatDate.js

```diff
@@ -15,8 +15,9 @@ const DATE_FORMAT_SHORT_YEAR = "DD-MMM-YY";
  * @param   {string | unknown} date - date to be formatted
  * @returns {string}                - formatted date
  */
-const formatDate = (date) =>
-  date && dayjs(date).tz("America/Vancouver").format(DATE_FORMAT_SHORT_YEAR);
+
+//'Africa/Accra' sets the time for the date pickers to UTC Time.  Purpose for this is because Openshift is on UTC where this app is currently running from as per 04/25/2024
+const formatDate = (date) => date && dayjs(date).tz("Africa/Accra").format(DATE_FORMAT_SHORT_YEAR);
```

Comment:
are your sure you just can't put "UTC" in here instead of "Africa/Accra"

Reply:
added better solution